### PR TITLE
Issue #43: Make accodrion title tag configurable

### LINKED ISSUE #53: Make accodrion title tag configurable

### PR DESCRIPTION
**Pull request by dench0**
Closes #53

### CODE REVIEW

File: plugins/accordion/views_bootstrap_accordion_plugin_style.inc [hunk 1]
```diff
@@ -14,6 +14,7 @@ class views_bootstrap_accordion_plugin_style extends views_plugin_style {
   public function option_definition() {
     $options = parent::option_definition();
     $options['title_field'] = array('default' => NULL);
+    $options['title_tag'] = array('default' => 'h3');
```
Comment (ericvanderwal) [inline plugins/accordion/views_bootstrap_accordion_plugin_style.inc]: I think the current default is H4? So maybe we should change this to H4 instead, to not create any breaking changes for sites that update?

Comment (dench0) [reply plugins/accordion/views_bootstrap_accordion_plugin_style.inc]: fixed

File: plugins/accordion/views_bootstrap_accordion_plugin_style.inc [hunk 2]
```diff
@@ -39,6 +40,23 @@ class views_bootstrap_accordion_plugin_style extends views_plugin_style {
...
+          'h2' => 'H2',
+          'h3' => 'H3',
+          'h4' => 'H4',
+          'h5' => 'H5',
+          'p' => 'P',
+          'div' => 'DIV',
+        ),
+        '#required' => TRUE,
+        '#default_value' => $this->options['title_tag'],
+        '#description' => t('Select the title tag.'),
```
Comment (klonos) [inline plugins/accordion/views_bootstrap_accordion_plugin_style.inc]: Suggestion:

```suggestion
        '#description' => t('HTML tag to be used for the title.'),
```

File: plugins/accordion/views_bootstrap_accordion_plugin_style.inc [hunk 3]
```diff
@@ -39,6 +40,23 @@ class views_bootstrap_accordion_plugin_style extends views_plugin_style {
         '#default_value' => $this->options['title_field'],
         '#description' => t('Select the field that will be used as the title.'),
       );
+
+      $form['title_tag'] = array(
+        '#type' => 'select',
+        '#title' => t('Title tag'),
```
Comment (klonos) [inline plugins/accordion/views_bootstrap_accordion_plugin_style.inc]:
```suggestion
        '#title' => t('Title HTML tag'),
```

**Comment by stpaultim** [comment]
Please open an issue as well. We prefer it if every PR is linked to specific issue. 

https://github.com/backdrop-contrib/views_bootstrap/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc

File: plugins/accordion/views_bootstrap_accordion_plugin_style.inc (hunk 2 above)
Comment (dench0) [reply plugins/accordion/views_bootstrap_accordion_plugin_style.inc]: fixed

File: plugins/accordion/views_bootstrap_accordion_plugin_style.inc (hunk 3 above)
Comment (dench0) [reply plugins/accordion/views_bootstrap_accordion_plugin_style.inc]: fixed

**Comment by stpaultim** [comment]
This looks good to me and has had three reviewers already.

**Pull request merged**